src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
+import core.basesyntax.machines.Bulldozer;
+import core.basesyntax.machines.Excavator;
+import core.basesyntax.machines.Machine;
+import core.basesyntax.machines.Truck;
+
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[] {
+                new Truck(),
+                new Bulldozer(),
+                new Excavator()
+        };
+
```

Comment:
Please don’t add redundant empty lines to your code

Reply:
Oh gosh, it seems that I missed these few ones. Sorry. Now it's fixed.